app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+
```

Comment:
You should check, that `parts` contains 3 elements and first part of command is `mv`